dev-infra/release/publish/pull-request-state.ts

```diff
@@ -17,11 +20,15 @@ export async function getPullRequestState(api: GitClient, id: number): Promise<P
   const {data} = await api.github.pulls.get({...api.remoteParams, pull_number: id});
   if (data.merged) {
     return 'merged';
-  } else if (data.closed_at !== null) {
+  }
+  // Check if the PR was closed more than 30 seconds ago, this extra time gives Github time to
+  // update the closed pull request to be associated with the closing commit.
+  // Note: a Date constructed with `null` creates an object at 0 time, which will never be greater
+  // than the current date time.
+  if (new Date(data.closed_at).getTime() + THIRTY_SECONDS_IN_MS > new Date().getTime()) {
```

Comment:
This condition is not correct. It will be true only if _not_ more than 30s have passed since `closed_at`. I think you were going for something like:

```suggestion
  if ((data.closed_at !== null) && (new Date(data.closed_at).getTime() + THIRY_SECONDS_IN_MS < Date.now())) {
```